FirebaseDynamicLinks/README.md

```diff
@@ -0,0 +1,25 @@
+# Firebase Dynamic Links SDK for iOS
+
+Firebase Dynamic Links are links that work the way you want, on multiple platforms, and whether or not your app is already installed.
```

Comment:
Rephrase this to be more clear and less market-y.

"Firebase Dynamic Links are universal deep links that persist across app installs."